src/features/mixer.py

```diff
@@ -58,6 +67,20 @@ def __next__(self):
     def __iter__(self):
         return self
 
+    def get_batch(self, batchsize=32):
+
+        batches = ()
+        for iterator in self.iterators:
+            batches += (iterator.get_batch(batchsize),)
+
+        if self.mix_method=='sum' or self.mix_method=='add':
+            mixed = np.sum( ibatch[1] for ibatch in batches )
```

Comment:
Maybe a comment assuring the reader that this only sums over the combination axis and not down to a scalar? Also, I think I'm getting confused by the named, but how is this differentiating between summing within examples (good) and over examples (not good)?